.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 - fluxbox &
 - wget http://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar
   -O selenium-server-standalone.jar
-- java -jar selenium-server-standalone.jar > /dev/null 2>/dev/null &
+- java -jar selenium-server-standalone.jar > selenium.log &
```

Comment:
I'm not sure if you need to specify both.  If you tested it though, that's fine

java -jar selenium-server-standalone.jar > selenium.log 2>selenium.log &